imports/styles/app.css

```diff
@@ -56,9 +39,10 @@ body {
 .reveal {
   width: fit-content;
 }
-.button.small {
-  margin-bottom: 0rem;
+.fa-trash-alt:hover {
+  color: red;
 }
-.button.cancel {
-  margin-bottom: 0rem;
+svg {
```

Comment:
like that, it's a global selection. what if you have an icon on the button your Add Player form - > e.g. a little "floppy disk" icon in front of the text on the button? suddenly it would have the same spacing and your button gets huge.

scope it by making the spacing around the svgs only applicable for the elements (in this case the divs) that you want